package chap10.lecture.multipleCatch;

public class SuperExceptionSample {
	public static void main(String[] args) {
		try {
			String a = "a100";
			int b =  Integer.parseInt(a);
		}catch(Exception e) {
			System.out.println("오류 발생");
		}
		
		System.out.println("프로그램 종료");
	}

}
